include/contribution.h

```diff
@@ -1 +1,13 @@
-//header file for contribution.c source file
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
When you merge, you should pick one of the two versions in a conflict. Besides, the header guards do not follow the convention. See convention list on CULearn.

Reply:
this guards were inserted while working on VS code.
They should be erased in new commit